plugins/controlboard/src/ControlBoardDriverCoupling.cpp

```diff
@@ -1330,7 +1367,7 @@ double HandMk5CouplingHandler::evaluateCoupledJointThumb(const double& q1)
      * The value of q1 is subtracted from the result as evaluateCoupledJoint
      * provides the absolute angle of the coupled distal joint with respect to the palm.
      */
-    return evaluateCoupledJoint(q1, mParamsThumb) - q1;
+    return evaluateCoupledJoint(q1, mFingerParameters[0]) - q1;
 }
 
 double HandMk5CouplingHandler::evaluateCoupledJointIndex(const double& q1)
```

Comment:
Given that we have decided to use the parameters of all the fingers, can we have a map `mMap` from finger name to the index accessing `mFingerParameters` such that we can call `evaluateCoupledJoint` with an interface like

```
double HandMk5CouplingHandler::evaluateCoupledJoint(const double& q1, const std::string& finger_name)
```

and inside the function we do

```
const FingerParameters& params = mFingerParameters.at(mMap.at(finger_name))
```
?

This way we could drop all the auxiliary functions `evalauteCoupledJoint{Thumb, Index, Pinky}`. The same would apply to `evaluatedCoupledJointJacobian`. Thanks!